packages/ses/src/module-load.js

```diff
@@ -72,18 +77,24 @@ const loadRecord = async (
     resolvedImports,
   });
 
+  for (const fullSpecifier of values(resolvedImports)) {
+    // Behold: recursion.
+    // eslint-disable-next-line no-use-before-define
+    const loadDependency = memoizedLoadWithErrorAnnotation(
+      compartmentPrivateFields,
+      moduleAliases,
+      compartment,
+      fullSpecifier,
+      queue,
+      loading,
+    );
+    // Queuing does not work well with unhandled rejection warnings.
+    promiseCatch(loadDependency, noop);
+    setAdd(queue, loadDependency);
```

Comment:
perhaps better to at least log the error

Reply:
one thing i do in these situations is throw an error at the start of a next tick. not great, not terrible

Reply:
in this case, the error is guaranteed to percolate to the drain and returned by `load`, unless it’s obscured by an earlier error. I could alternately gather an aggregate error where it’s currently draining, but the important effect is the same: `load` returns a promise that’s eventually rejected.

Reply:
I definitely do not want to elevate a load failure to an uncaught exception. The caller should be able to handle any exception effected by the work scheduled by `load`. I’m satisfied with this as-is, but I will consider ways to ensure that every failure gets propagated.

Reply:
Here’s my proposed solution for error aggregation:
https://github.com/endojs/endo/pull/877/commits/ea4a990f89353c3903c80340e929980af135658b